lab-examples/clos02/setup.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+CFG_DIR=./configs
+
+configure_SRL () {
+  OUT=$(gnmic -a clab-clos02-$1 --timeout 30s -u admin -p admin -e json_ietf --skip-verify set --update-path / --update-file $CFG_DIR/$1.yaml 2>&1)
+  echo $OUT | grep -q -e '\"operation\": \"UPDATE\"'
+  if [ $? -eq 0 ]; then
+    SAVE=$(docker exec -it clab-clos02-$1 sr_cli "save startup" 2>&1)
+  else
+    echo "Error: Unable to push config. Check if NE is reachable with gRPC enabled."
```

Comment:
I think here it makes sense to show the `$OUT` as well, as in the case of a failure it is interesting to see what went wrong

Reply:
$OUT does not have any valid output. So push it into /dev/null.